portal-web/test/functional/com/liferay/portalweb/tests/enduser/staging/pgstaging/CPRolesPGStaging.testcase

```diff
@@ -39,251 +39,263 @@ definition {
 	test AccessStaging {
 		property portal.acceptance = "true";
 
-		JSONStaging.enableLocalStaging(groupName = "Site Name");
+		task ("Given a regular user is added") {
```

Comment:
For better readability, we used to separate the GWT words with a colon ":", like

"Given: A regular user is added"
or
"Given: We add a new regular user"

Think of them as prefixes, rather than actual parts of the sentences.
The sentences should reflect what the Persona (Instance admin or End-User) wants. Try to define the sentences as you are the End User, and thing what you want to achieve in this step :)

cc @JoyceWang08211 , @weller0246